src/ucp/wireup/wireup.c

```diff
@@ -726,40 +726,37 @@ ucp_wireup_connect_lane_to_iface(ucp_ep_h ep, ucp_lane_index_t lane,
                                  ucp_worker_iface_t *wiface,
                                  const ucp_address_entry_t *address)
 {
-    ucp_lane_index_t proxy_lane = ucp_ep_get_proxy_lane(ep, lane);
     uct_ep_params_t uct_ep_params;
     uct_ep_h uct_ep;
     ucs_status_t status;
 
     ucs_assert(wiface->attr.cap.flags & UCT_IFACE_FLAG_CONNECT_TO_IFACE);
 
-    if ((ep->uct_eps[lane] == NULL) || ucp_wireup_ep_test(ep->uct_eps[lane])) {
-        if ((proxy_lane == UCP_NULL_LANE) || (proxy_lane == lane)) {
-            /* create an endpoint connected to the remote interface */
-            ucs_trace("ep %p: connect uct_ep[%d] to addr %p", ep, lane,
-                      address);
-            uct_ep_params.field_mask = UCT_EP_PARAM_FIELD_IFACE      |
-                                       UCT_EP_PARAM_FIELD_DEV_ADDR   |
-                                       UCT_EP_PARAM_FIELD_IFACE_ADDR |
-                                       UCT_EP_PARAM_FIELD_PATH_INDEX;
-            uct_ep_params.iface      = wiface->iface;
-            uct_ep_params.dev_addr   = address->dev_addr;
-            uct_ep_params.iface_addr = address->iface_addr;
-            uct_ep_params.path_index = path_index;
-            status = uct_ep_create(&uct_ep_params, &uct_ep);
-            if (status != UCS_OK) {
-                /* coverity[leaked_storage] */
-                return status;
-            }
-
-            ucp_wireup_assign_lane(ep, lane, uct_ep, "");
-        }
+    if ((ep->uct_eps[lane] != NULL) && !ucp_wireup_ep_test(ep->uct_eps[lane])) {
+        /* Lane already exists */
```

Comment:
minor: lowercase

Reply:
actually, comments are supposed to start with uppercase

Reply:
I see